src/Components/Aspire.Keycloak/AspireKeycloakExtensions.cs

```diff
@@ -0,0 +1,95 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Provides extension methods for registering Keycloak-related services in an <see cref="IHostApplicationBuilder"/>.
+/// </summary>
+public static class AspireKeycloakExtensions
+{
+    private const string KeycloakBackchannel = nameof(KeycloakBackchannel);
+
+    /// <summary>
+    /// Adds Keycloak JWT Bearer authentication to the application.
+    /// </summary>
+    /// <param name="builder">The <see cref="AuthenticationBuilder" /> to add services to.</param>
+    /// <param name="connectionName">The connection name to use to find a connection string.</param>
```

Comment:
Name of the keycloak service and more details about the expected format and how we use it.

Reply:
Improved docs.